SwissDataScienceCenter/renku-ui
# Turn on the preview version banner

Fix #1373

# Testing

Connect to the deployment. 
- Click on the "Try out a preview of the new RenkuLab UI" link. You should be redirected to the new UI.
- Close your browser and come back to the URL. You should still see the new UI
- Delete the "other-ui" cookie and refresh. You should be back to the old UI.

# Appearance

![image](https://user-images.githubusercontent.com/1196411/121018206-78c70080-c79e-11eb-91bc-226e63ec7740.png)


/deploy
